source/javascripts/components/StepItem/StepItem.tsx

```diff
@@ -20,6 +19,7 @@ type StepItemProps = {
 	version: string;
 	isDeprecated: boolean;
 	isVerified: boolean;
+	isOfficial: boolean;
```

Comment:
`isOfficial` is defined here, but was never used, since you are calling `step.isOfficial()` instead.
A couple of lines later, you omit the `isVerified` parameter from the props to use `step.isVerified()`, which is good.
However I would make `isDeprecated`, `isVerified` and ,`isOfficial` to be passed in and used in the same manner,
meaning:
 - remove from `StepItemProps`
 - remove from the step invocation (as you did with `isVerified`)
 - use `step.isVerified()`, `step.isOfficial()` and `step.isDeprecated()` methods instead of incoming properties.
 - remove from r-step-item attributes form the slim file